lib/services/networkManagement2/generate.cmd

```diff
@@ -0,0 +1,17 @@
+::
+:: Microsoft Azure SDK for Net - Generate library code
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+
+@echo off
+set autoRestVersion=0.13.0-Nightly20151115
```

Comment:
`set autoRestVersion=0.13.0-Nightly20151126` This will resolve the issue of reserved words in long running operation names as pointed out by Roman in email conversation.